STANDARD.md

```diff
@@ -424,11 +424,29 @@ meshes):
     - example: `(0.0, 100.0, 0.0)` or `(0.5, 0.5, 0.5)`
 
   - `gridUnitSI`
-    - type: *(float64 / REAL8)*
+    - type: 1-dimensional array containing N *(float64 / REAL8)*
+            elements, where N is the number of dimensions in the simulation
     - description: unit-conversion factor to multiply each value in
                    `gridSpacing` and `gridGlobalOffset`, in order to convert
                    from simulation units to SI units
-    - example: `1.0e-9`
+    - example: `(1.0e-9, 1.0e-9, 1.0e-6)`
+
+  - `gridUnitDimension`
+    - type: array of 7 N *(float64 / REAL8)*
```

Comment:
I would write `N arrays of 7 *(float64 / REAL8)*`

Reply:
Ok, that's one possibility.
However, the thing is that it is stored as a single 1D array.

Reply:
BTW, I see a few REAL8 mentioned, but this word isn't actually standard (Neither is real(8)) in Fortran. Fortran 2008 provides standard kinds: REAL32, REAL64, REAL128
http://fortranwiki.org/fortran/show/Real+precision

Reply:
Huh, very good point. This is also something for #187

Reply:
>  However, the thing is that it is stored as a single 1D array.

Ah I see, ok with me.

Reply:
My thought would be to not even mandate any precision (ie just say "real"). For example, why should the standard disallow the use of real4?

Reply:
@RemiLehe pls replace `array` with `1-dimensional array` for consistency.